scripts/start_arm64_local_marqo_os.sh

```diff
@@ -1,23 +1,43 @@
 #!/bin/bash
 # args:
 # $1 : marqo_image_name - name of the image you want to test
+# $@ : env_vars - strings representing all args to pass docker call
 
 export LOCAL_OPENSEARCH_URL="https://localhost:9200"
 
-docker rm -f marqo-os &&
-    docker run --name marqo-os -p 9200:9200 -p 9600:9600 -e "discovery.type=single-node" marqoai/marqo-os:0.0.3-arm &
+docker rm -f marqo-os
```

Comment:
modeled this after `start_local_marqo_os.sh` because the current way encounters errors with start marqo-os & hanging after restart